### PR TITLE
New product header

### PR DESCRIPTION
**Pull request by poulch**
I want to merge this change because...

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
APPS_MARKETPLACE_API_URI=https://apps.staging.saleor.io/api/v2/saleor-apps
SALEOR_APPS_ENDPOINT=https://apps.saleor.io/api/saleor-apps

### Do you want to run more stable tests?

To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] giftCard
3. [ ] category
4. [ ] collection
5. [ ] attribute
6. [ ] productType
7. [ ] shipping
8. [ ] customer
9. [ ] permissions
10. [ ] menuNavigation
11. [ ] pages
12. [ ] sales
13. [ ] vouchers
14. [ ] homePage
15. [ ] login
16. [ ] orders
17. [ ] products
18. [ ] app
19. [ ] plugins
20. [ ] translations
21. [ ] navigation
22. [ ] variants
23. [ ] payments

CONTAINERS=1

### CODE REVIEW

**Review by krzysztofzuraw — APPROVED** [review]

File: src/components/AppLayout/FilterBar/Filter.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      menu.reduce((acc, filterElement) => {
+        const dataFilterElement = data.find(
+          ({ name }) => name === filterElement.name,
+        );
+
+        if (!dataFilterElement) {
+          return acc;
+        }
+
+        return acc + (dataFilterElement.active ? 1 : 0);
```
Comment (andrzejewsky) [inline src/components/AppLayout/FilterBar/Filter.tsx:50]: **suggestion**: it deserves dedicated function, hard to read what this memo is doing

File: src/components/AppLayout/FilterBar/styles.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      backgroundColor: vars.colors.background.surfaceNeutralPlain,
+      overflowY: "scroll",
+      boxShadow: `0px 6px 11px 9px ${vars.colors.background.surfaceNeutralHighlight}`,
+      height: 450,
+      width: 376,
+      zIndex: 5,
+    },
+  }),
+  { name: "Filter" },
+);
```
Comment (andrzejewsky) [inline src/components/AppLayout/FilterBar/styles.ts:16]: **issue**: please don't use old macaw anymore in the newly created stuff, style with box or sprinkles

File: src/products/components/ProductListTiles/ProductListTiles.tsx [hunk 3]
```diff
@@ -40,11 +51,21 @@ export const ProductListTiles: React.FC<ProductListTilesProps> = ({
...
-        <Box display="flex" justifyContent="center" height="100%" marginY={12}>
-          <CircularProgress />
-        </Box>
-      )}
+      );
+    }
+
+    return (
+      <Box padding={9}>
+        <Typography align="center">
```
Comment (andrzejewsky) [inline src/products/components/ProductListTiles/ProductListTiles.tsx:59]: **issue**: use our Text component

File: src/products/components/ProductListDatagrid/ProductListDatagrid.tsx [hunk 4]
```diff
@@ -221,53 +223,55 @@ export const ProductListDatagrid: React.FC<ProductListDatagridProps> = ({
...
+              query={columnQuery}
+              onQueryChange={onColumnQueryChange}
+              onSave={handleColumnChange}
+            />
+          )}
         />
-      </div>
-    </DatagridChangeStateContext.Provider>
+
+        <div className={sprinkles({ paddingX: 9 })}>
```
Comment (andrzejewsky) [inline src/products/components/ProductListDatagrid/ProductListDatagrid.tsx:265]: **suggestion**: Box

**Review by andrzejewsky — CHANGES_REQUESTED** [review]

**Review by andrzejewsky — APPROVED** [review]

**Pull request merged**